docs/appendices/release-notes/unreleased.rst

```diff
@@ -57,6 +57,9 @@ None
 Fixes
 =====
 
+- Fixed an issue in :ref:`ref-values` that would not allow combining expressions
```

Comment:
I was not sure whether expressions implicit cast are more of a new feature or a fix. I am more inclined to have it as a fix on the same line with `null`.